ui/src/app/views/concept-set-details.tsx

```diff
@@ -57,15 +57,14 @@ const ConceptSetMenu: React.FunctionComponent<{
     content={ <React.Fragment>
       <TooltipTrigger content={<div>Requires Write Permission</div>}
                       disabled={canEdit}>
-        <MenuItem icon='pencil'
-                  onClick={() => onEdit}
+        <MenuItem onClick={() => onEdit}
                   disabled={!canEdit}>
```

Comment:
can probs put this on the same line as `MenuItem` now. it's a little cleaner

Reply:
nice!